test/env

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -e
+
+
+exec {BASH_XTRACEFD}>&1
+echo "$0" "$@"
+set -x
+
+
+if [ -z "${MPV_MPRIS_TEST_PLUGIN+set}" ] ; then
+	export MPV_MPRIS_TEST_PLUGIN=../mpris.so
+fi
+
+if [ -z "$MPV_MPRIS_TEST_PLAY" ] ; then
+	export MPV_MPRIS_TEST_PLAY=/usr/share/sounds/freedesktop/stereo/alarm-clock-elapsed.oga
+fi
+
+if [ -z "$MPV_MPRIS_TEST_MPV_IPC" ] ; then
+	export MPV_MPRIS_TEST_MPV_IPC=.
+fi
+
+if [ -z "$MPV_MPRIS_TEST_LOG" ] ; then
+	export MPV_MPRIS_TEST_LOG=.
+fi
+
+if [ -z "$MPV_MPRIS_TEST_DBUS" ] ; then
+	MPV_MPRIS_TEST_DBUS=.
+fi
+
+if [ -z "$MPV_MPRIS_TEST_XAUTH" ] ; then
+	MPV_MPRIS_TEST_XAUTH=.
+fi
+
+if [ -z "$MPV_MPRIS_TEST_TMP" ] ; then
+	MPV_MPRIS_TEST_TMP=.
+fi
+
+if [ -n "$MPV_MPRIS_TEST_PLUGIN" ] ; then
+	if [ ! -f "$MPV_MPRIS_TEST_PLUGIN" ] ||
+	   [ ! -r "$MPV_MPRIS_TEST_PLUGIN" ] ||
+	   [ ! -x "$MPV_MPRIS_TEST_PLUGIN" ] ; then
+		echo "$MPV_MPRIS_TEST_PLUGIN not an existing file with rx perms" >&2
+		exit 1
+	fi
+fi
+
+if [ ! -f "$MPV_MPRIS_TEST_PLAY" ] ||
+   [ ! -r "$MPV_MPRIS_TEST_PLAY" ] ; then
+	echo "$MPV_MPRIS_TEST_PLAY not an existing readable file" >&2
+	exit 1
+fi
+
+writable_dir () {
+	if [ ! -d "$1" ] ||
+	   [ ! -r "$1" ] ||
+	   [ ! -w "$1" ] ||
+	   [ ! -x "$1" ] ; then
+		echo "$1 not an existing writable directory" >&2
+		exit 1
+	fi
+}
+
+writable_dir "$MPV_MPRIS_TEST_MPV_IPC"
+writable_dir "$MPV_MPRIS_TEST_DBUS"
+writable_dir "$MPV_MPRIS_TEST_XAUTH"
+writable_dir "$MPV_MPRIS_TEST_LOG"
+writable_dir "$MPV_MPRIS_TEST_TMP"
+
+
+export -n \
+	MPV_MPRIS_TEST_DBUS \
+	MPV_MPRIS_TEST_XAUTH \
+	MPV_MPRIS_TEST_TMP
+
+
+exec \
+env \
+MPV_MPRIS_TEST_NAME="$1" \
+XDG_RUNTIME_DIR="$MPV_MPRIS_TEST_DBUS" \
+TEMPDIR="$MPV_MPRIS_TEST_TMP" \
+TMPDIR="$MPV_MPRIS_TEST_TMP" \
+TEMP="$MPV_MPRIS_TEST_TMP" \
+TMP="$MPV_MPRIS_TEST_TMP" \
+dbus-run-session -- \
+xvfb-run --auto-servernum --auth-file "$MPV_MPRIS_TEST_XAUTH/$(basename "$1").Xauthority" \
```

Comment:
Replacing this with -f does indeed work in arch linux. I've reported the bug upstream.

Reply:
Fixed, added a comment about it.

What is the bug link?

Reply:
https://bugs.archlinux.org/task/73865

Reply:
Added to the patch. Looks like it is fixed now so maybe I can switch back to the long option though?